### PR TITLE
refactor(switch): separate css vars

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/1158

### CODE REVIEW

File: src/patternfly/components/Switch/switch.scss [hunk 1]
```diff
@@ -85,8 +87,8 @@
 
     &::before {
       position: absolute;
-      top: var(--pf-c-switch__toggle--Padding);
-      left: var(--pf-c-switch__toggle--Padding);
```
Comment (andybraren) [inline src/patternfly/components/Switch/switch.scss]: `--pf-c-switch__toggle--Padding` at the top is no longer needed now that it's split.

File: src/patternfly/components/Switch/switch.scss [hunk 2]
```diff
@@ -19,6 +19,8 @@
   --pf-c-switch__toggle--before--BorderRadius: var(--pf-c-switch__toggle--before--Width);
   --pf-c-switch__toggle--before--BoxShadow: var(--pf-global--BoxShadow--md);
   --pf-c-switch__toggle--before--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-switch__toggle--before--Top: calc(#{var(--pf-c-switch--Height)} / 2 * (1 - 1 / #{var(--pf-global--golden-ratio)}));
+  --pf-c-switch__toggle--before--Left: var(--pf-c-switch__toggle--before--Top);
   --pf-c-switch__label--Color: var(--pf-global--Color--dark-100);
```
Comment (andybraren) [inline src/patternfly/components/Switch/switch.scss]: The variables for at least `--pf-c-switch__toggle` and `--pf-c-switch__label` could be reordered to match the order in which they're used down below, but that's not really important (or this PR's job).

File: src/patternfly/components/Switch/switch.scss (hunk 1 above)
Comment (mcoker) [reply src/patternfly/components/Switch/switch.scss]: good catch!

File: src/patternfly/components/Switch/switch.scss (hunk 2 above)
Comment (mcoker) [reply src/patternfly/components/Switch/switch.scss]: Ooh fancy! I don't think we do that as a thing (or I haven't anyways), but sure why not?

**Comment by mcoker** [comment]
That's a big 10-4 good buddy. Updated. Over and out.

**Pull request merged**